contracts/amm/AmmDataProvider.sol

```diff
@@ -262,4 +277,289 @@ contract AmmDataProvider is IAmmDataProvider {
 
         return collateralAmount;
     }
+
+    /// @dev Calculate the collateral amount receivable by redeeming the given
+    /// Series' bTokens and wToken
+    /// @param seriesId The index of the Series
+    /// @param wTokenBalance The wToken balance for this Series owned by this AMM
+    /// @param bTokenBalance The bToken balance for this Series owned by this AMM
+    /// @return The total amount of collateral receivable by redeeming the Series' option tokens
+    function getRedeemableCollateral(
+        uint64 seriesId,
+        uint256 wTokenBalance,
+        uint256 bTokenBalance
+    ) private view returns (uint256) {
+        uint256 unredeemedCollateral = 0;
+        if (wTokenBalance > 0) {
+            (uint256 unclaimedCollateral, ) = seriesController.getClaimAmount(
+                seriesId,
+                wTokenBalance
+            );
+            unredeemedCollateral += unclaimedCollateral;
+        }
+        if (bTokenBalance > 0) {
+            (uint256 unexercisedCollateral, ) = seriesController
+                .getExerciseAmount(seriesId, bTokenBalance);
+            unredeemedCollateral += unexercisedCollateral;
+        }
+
+        return unredeemedCollateral;
+    }
+
+    /// @notice Calculate the amount of collateral the AMM would received if all of the
+    /// expired Series' wTokens and bTokens were to be redeemed for their underlying collateral
+    /// value
+    /// @return The amount of collateral token the AMM would receive if it were to exercise/claim
+    /// all expired bTokens/wTokens
+    function getCollateralValueOfAllExpiredOptionTokens(
+        uint64[] memory openSeries,
```

Comment:
I think we do need MinterAmm.getAllSeries, because the `MinterAmm.openSeries` contract variable is private and so we need a public getter for it. In particular that getter needs to create a copy of the underlying `uint64's` so that a contract which gets a reference `openSeries` is not able to alter the Amm's `openSeries`.

How would it work if we added getAllSeries to the AmmDataProvider? The AmmDataProvider doesn't have an `openSeries` variable so it wouldn't be able to get it.

Reply:
Ahhh, I was thinking we could possibly move openSeries into the AmmDataProvider, however I think I was misinterpreting the purpose of the DataProvider I looked again and it isnt holding any state. We are good to keep MinterAmm.getAllSeries. I will resolve this. 

Thanks for explaining!